Dryad Digital Repository.js

```diff
@@ -0,0 +1,686 @@
+{
+	"translatorID": "7a81d945-7d9c-4f8c-bd7b-4226c1cab40e",
+	"label": "Dryad Digital Repository",
+	"creator": "Nathan Day",
+	"target": "^https?://(?:www\\.)?datadryad\\.org/",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-06-22 11:24:00"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2014 Dryad Digital Repository
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+var doiUrlBase      = 'http://dx.doi.org/';
+var packageXpath    = '//meta[@name="DCTERMS.isPartOf"]/@content';
+var multiplesXpath  = '//li[contains(@class,"ds-artifact-item")]/div/a';
+var doiXpath        = '//meta[@name="DC.identifier"][starts-with(@content,"doi:10.5061/dryad.")]/@content';
+var dcTypeXpath     = '//meta[@name="DC.type"][1]/@content';
+var discoverPage    = /^https?:\/\/(www\.)?datadryad\.org\/discover/;
+var urlToDoi        = /^.*?doi:/;
+
+function detectWeb(doc, url) {
+	// Dryad search page
+	if (url.match(discoverPage)) {
```

Comment:
`RegExp.prototype.test` is preferred where you don't actually need the results of the match